signing-scheme.md

```diff
@@ -0,0 +1,97 @@
+This is supported through the the Timestamp signature unsigned attribute in the signature envelope. Currently Notary V2 uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.
+
+* `notary.x509.signingAuthority` MUST use a timestamp attribute that is generated by a signing service as part of the original signature itself to determine authenticated signing time (timestamp).
+This is supported through the *Authentic Signing time* attribute in the signature envelope.
+
+## Signature Verification
+
+Signature verification requires that a `Signing Scheme` attribute is present in the signature and it’s treated as critical i.e. the attribute MUST be understood and processed by the verifier.
+For the JWS signature format, the attribute name is `io.cncf.notary.signingScheme` , with supported values `notary.x509` and `notary.x509.signingAuthority`.
+Any other value will fail signature verification i.e when Notary supports an additional Signing Scheme, clients (like *notation*) MUST be updated to a version that supports the new signing scheme.
+
+### Trust Stores
+
+Each *Signing Scheme* defines the set of trust store types (e.g. CA) that it uses for signature verification.
+
+`notary.x509`
+
+* Uses trusts store types Certificate Authority (CA) and Timestamping Authority (TSA) during signature verification.
+The signature is verified against the trust store of type CA, and the *Timestamp signature* is verified against the trust store of type TSA (to determine the signing time).
+* For signature verification to be successful
+  * The verifying entity’s trust store MUST contain the trusted root certificates under named trust stores of type CA (`{CONFIG}/notation/truststore/x509/ca`) and TSA(`{CONFIG}/notation/truststore/x509/tsa`)
+  * The named trust stores MUST be specified in *trustpolicy.json*. E.g. *trustPolicy.trustStores* with value of `ca:acme-rockets,tsa:acme-tsa`.
+
+`notary.x509.signingAuthority`
+
+* Uses trusts store type Signing Authority during signature verification.
+The signature is verified against the trust store of type Signing Authority.
+The signing time is determined using the *Authentic Signing time* attribute in the signature envelope, and does not rely on a seperate TSA generated Timestamp signature.
+* For signature verification to be successful
+  * The verifying entity’s trust store MUST contain the trusted root certificates under named trust stores of type CA (`{CONFIG}/notation/truststore/x509/signingAuthority`).
+  * The named trust stores MUST be specified in *trustpolicy.json*. E.g. *trustPolicy.trustStores* with value of `signingAuthority:foobar` .
+
+## FAQ
+
+*Q:* What is the relationship of Signing Scheme with Signature Envelope format?
+*A:* Signing Scheme aims to be agnostic of the Signature Envelope format.
+A given signing scheme can be implemented through any signature envelope format (such as JWS or COSE) as long as it can support the required signature schema used by the signing scheme.
```

Comment:
in rich diff both Q and A are in same line.

Reply:
mardown tip: double spaces at the end of line 87 will make line 88 indented

Reply:
Addressed